lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -1999,11 +1999,22 @@ galaxy:
   # for production servers yet.
   #local_task_queue_workers: 2
 
-  # Enable job recovery (if Galaxy is restarted while cluster jobs are
-  # running, it can "recover" them when it starts).  This is not safe to
-  # use if you are running more than one Galaxy server using the same
-  # database.
-  #enable_job_recovery: true
```

Comment:
For anyone wondering, the last use of `enable_job_recovery` was removed in commit c1b0fab1e8f22a9d57d68b53c41ee706469f5b65 .